### PR TITLE
Project: awesomebooks basic-ui

### PR DESCRIPTION
**Pull request by wwwdrt**
### 🎯Learning objectives

- Understand different ways to create objects in JavaScript.
- Create and access properties and methods of JavaScript objects.

### 📃Project requirements

- [x] Implement only a basic UI with plain HTML.
- [x] Create a collection that keeps a list of books (hint: you can use an array of objects for that).
- [x] Create a function to add a new book to the collection, with title and author.
- [x] Create a function to remove a book from the collection (hint: you can use the [array filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method).
- [x] Display all books saved in the collection in the top part of the page.
- [x] Make sure that when a user clicks the "Add" button:
- [x] A new book is added to the collection.
- [x] The new book is displayed in the page.
- [x] Make sure that when a user clicks the "Remove" button:
- [x] The correct book is removed from the collection.
- [x] The correct book dissapears from the page.
- [x] Make sure that data is preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md).
- [x] To debug your application, you can use [console.log()](https://developer.mozilla.org/en-US/docs/Web/API/Console/log). Remember to remove debugging code before pushing your code to the remote repository.
- [x] Don't use any library or framework, use only plain JavaScript.

### 📝Notes

👋Hey, we are looking forward to your feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,19 +218,16 @@ Feel free to check the [issues page](../../issues/).
...
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
+> Thanks to Microverse for the amazing curriculum.
 
-I would like to thank...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (zieeco) [inline README.md:233]: ## ❓ FAQ (OPTIONAL)

- Hey, as this section of the readme is optional, kindly remove it or add some questions and answers as this should not be left empty to make your readme more professional.

Comment (zieeco) [reply README.md:233]: ## 📝 License 

- Hey, let's kindly remove this placeholder text instructing you on how important it is to add a license to make your readme more professional

```
_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

File: script.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                    <li>${book.author}</li>
+                    <button type="button" id="remove-book">REMOVE</button>
+                </ul>
+            `;
+      libraryContainer.appendChild(bookContainer);
+
+      const removeButton = bookContainer.querySelector('#remove-book');
+      removeButton.addEventListener('click', () => {
+        library.splice(index, 1);
+        // eslint-disable-next-line no-use-before-define
```
Comment (zieeco) [inline script.js:42]: **[OPTIONAL]**

- Hey, as this is a bad practice disabling a linter rule, I'm ignoring this now as this is your first project on javascript but this won't be the case for subsequent projects. let's try to write code that conforms to the linter to improve our development skills.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+  form.addEventListener('submit', (event) => {
+    event.preventDefault();
+    const book = {
+      title: title.value,
+      author: author.value,
+    };
+
+    library.push(book);
+    // eslint-disable-next-line no-use-before-define
```
Comment (zieeco) [inline script.js:63]: **[OPTIONAL]**

- Hey, as this is a bad practice disabling a linter rule, I'm ignoring this now as this is your first project on javascript but this won't be the case for subsequent projects. let's try to write code that conforms to the linter to improve our development skills.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good commit messages :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
*Check the comment under the review*


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @team, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**